HISTORY.rst

```diff
@@ -2,6 +2,10 @@
 History
 =======
 
+0.7.4
+-----
+* CI: Add GitHub Actions workflow to check if HISTORY.rst is modified in a pull request to force writing history during development (PR #873)
```

Comment:
```suggestion
[Unreleased]
-------------

Added
^^^^^
* CI: Add GitHub Actions workflow to check if HISTORY.rst is modified in a pull request to force writing history during development (PR #873)

Changed
^^^^^^^

Deprecated
^^^^^^^^^

Removed
^^^^^^^

Fixed
^^^^

```

maybe we should follow the suggestions from https://keepachangelog.com/en/1.0.0/, so it would be sth like this. 
I skipped ``Security``, I think we dont have this kind of changes